### PR TITLE
Feature/hosting

### PR DESCRIPTION
**Pull request by danquah**
This PR implements kubernetes based hosting of Material-List.

It includes
- Action for building a release for every commit
- Action for pushing and deploying a release on commit to master
- Terraform configuration for provisioning a project
- Docker-setup for hosting the project, and a simple development setup

DML-16

### CODE REVIEW

File: infrastructure/docker/php-fpm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+FROM php:7.2-fpm-alpine
```
[the remark this answers is not in the document]
Comment (danquah) [reply infrastructure/docker/php-fpm/Dockerfile]: Prod currently runs on 7.3 - so it seems to be a viable way to go.

[the remark this answers is not in the document]
Comment (xendk) [reply infrastructure/docker/php-fpm/Dockerfile]: Bump it. :+1:

[the remark this answers is not in the document]
Comment (kasperg) [reply infrastructure/docker/php-fpm/Dockerfile]: See #27 @danquah.

[the remark this answers is not in the document]
Comment (kasperg) [reply infrastructure/docker/php-fpm/Dockerfile]: @danquah How about #27 then?

File: .github/actions/deployer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+ARG CLOUD_SDK_TAG=257.0.0-slim
```
Comment (achton) [inline .github/actions/deployer/Dockerfile]: We're already at 261.0.0 now :sob:

File: infrastructure/infrastructure.env [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Repository that hosts our docker images.
+DOCKER_REPO=eu.gcr.io/reload-material-list-3
```
Comment (achton) [inline infrastructure/infrastructure.env]: It's just words but technically it's a registry, no?

File: infrastructure/docker/php-fpm/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (danquah) [reply infrastructure/docker/php-fpm/Dockerfile]: 👍

File: infrastructure/infrastructure.env (hunk 3 above)
Comment (danquah) [reply infrastructure/infrastructure.env]: sure is :)

**Pull request merged**